Google Threat Intel Source (c2c) (#6028)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-threat-intel-source.md

---
id: google-threat-intel-source
title: Google Threat Intel Source
sidebar_label: Google Threat Intel
tags:
  - cloud-to-cloud
  - google-threat-intel

> description: Learn how to collect Varonis Alerts with their detailed information and send them to Sumo Logic.

Reviewer: ```suggestion
description: Learn how to collect Google Threat Intel data and send it to Sumo Logic.
```

---

import ForwardToSiem from '/docs/reuse/forward-to-siem.md';
import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/send-data/Google-Threat-Intel.png')} alt="icon" width="50"/>

> The Sumo Logic source for Google Threat Intel enables you to ingest the indicator data from Google Threat Intelligence API and send it to Sumo as STIX threat indicators. This integration elevates the value of threat hunting by providing tailored risk profiles, including actors, campaigns, and malware families, to enable proactive threat tracking and mitigation.

Reviewer: ```suggestion
The Sumo Logic source for Google Threat Intel enables you to ingest the indicator data from the Google Threat Intelligence API and send it to Sumo Logic as STIX threat indicators. This integration elevates the value of threat hunting by providing tailored risk profiles, including actors, campaigns, and malware families, to enable proactive threat tracking and mitigation.
```

## Data collected

| Polling Interval | Data |
| :--- | :--- |
| 1 hour |  [Indicators](https://gtidocs.virustotal.com/reference/get-objects-from-the-ioc-stream) |

## Setup

### Vendor configuration

The Google Threat Intel source requires you to provide the **API key** found in the [Threat Intelligence API](https://gtidocs.virustotal.com/docs/how-get-gti-api-keys#threat-intelligence-api) section in the Threat Intelligence portal.

### Source configuration

> When you create a Varonis Alerts Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).

Reviewer: ```suggestion
When you create a Google Threat Intel Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

> To configure Varonis Alerts Source:

Reviewer: ```suggestion
To configure the Google Threat Intel Source:
```

| config | JSON Object | [Configuration object](#configuration-object) | Yes | Source type specific values. |

### Configuration Object

| Parameter | Type | Required | Default | Description | Example |
|:--|:--|:--|:--|:--|:--|
| name | String | Yes | `null` | Type a desired name of the source. The name must be unique per Collector. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_source`. | `"mySource"` |
| description | String | No | `null` | Type a description of the source. | `"Testing source"`
| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
| apiKey | String | Yes | `null` | API key of your account. |  |

> | userSourceId | String | GoogleThreatIntelligence | `null` | The Sumo namespace in which the indicators will be stored. |  |

Reviewer: ```suggestion
| userSourceId | String | GoogleThreatIntelligence | `null` | The Sumo Logic namespace in which the indicators will be stored. |  |
```
